### DISCUSSION TITLE
Namespace Objects

### DISCUSSION THREAD

**Discussion by carlmontanari** [aptakube/aptakube#41] [Q&A] [answered]
Hey there, saw the reddit thread and decided to give aptakube a spin. Looks really nice and is super snappy!

I didn't see any way to look at namespace objects themselves, am I just missing something obvious, or are they hidden somewhere sneaky?

As a side thought/note -- I wonder if resources that don't have special handling (like nodes/deployments/etc.) could show up similar to how CRDs do? As in, right now there isn't (like I said, unless I'm blind!) pane for namespaces, or leases for example -- could there be a section for "general" resources that don't have the extra polish like deployments?

While I'm taking up space here, +1 to arbitrary kubeconfigs in arbitrary locations as well :)

**Comment by goenning** [comment, marked as answer]
Yeah, we don't support all built-in resources yet. It's an interesting idea to give a "generic view" to all these resources as that can be generated dynamically.

Let me try it for a bit to see if it's possible. 

In case it's not, are you missing something else other than namespaces?

**Reply by carlmontanari** [reply]
Awesome!

I think in order to pull the trigger and pay id have to have all the core bits (generic way would be more than acceptable) —- namespaces are the obvious one that jumps out to me but from time to time I end up looking at probably everything (I know that’s not helpful hah sorry!). For now I just stick with kubectl so I’m never limited, but if aptakube supported all the things I think I’d make that jump.

**Reply by goenning** [reply]
Mind if ask you how are you finding the current sidebar navigation with the current amount of built-in resources + CRD? 

The list will grow quite a bit after adding all built-in resources, so I'm thinking of ways of improving the navigation as it might get difficult to navigate. But this is just my perception of it, how do you feel about it currently?

**Reply by carlmontanari** [reply]
Yeah its working well so far -- w/ the search and collapsing each api group its been super manageable. I can definitely see it getting a bit out of hand though :) Perhaps a config option to selectively show/hide individual types or even full api groups? Even without that, with the api groups being collapsable I think it would be totally chill for my use case at least.
